iina.xcodeproj/project.pbxproj

```diff
@@ -523,6 +523,39 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		0507C25E2B5617650043AD05 /* he */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = he; path = he.lproj/MainMenu.strings; sourceTree = "<group>"; };
+		0507C25F2B5617660043AD05 /* he */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = he; path = he.lproj/MainWindowController.strings; sourceTree = "<group>"; };
```

Comment:
The `InfoPlist.strings` file is also missing from Xcode. The file is present, but is empty. I show the Italian file for comparison:
```text
low-batt@gag iina (he *$)$ cat he.lproj/InfoPlist.strings 
low-batt@gag iina (he *$)$ 
low-batt@gag iina (he *$)$ cat it.lproj/InfoPlist.strings 
/* (No Comment) */
"NSHumanReadableCopyright" = "Distribuito sotto licenza GPLv3";
low-batt@gag iina (he *$)$ 
```

Reply:
https://github.com/iina/iina/pull/4937

Reply:
@uiryuu I see you fixed it at https://github.com/iina/iina/pull/4937. Does it block this PR?